pyEpiabm/pyEpiabm/tests/test_utility/test_distance_metrics.py

```diff
@@ -0,0 +1,22 @@
+import unittest
+
+from pyEpiabm.utility import DistanceFunctions
+
+
+class TestDistanceFunctions(unittest.TestCase):
+    """Test the 'DistanceFunctions' class.
+    """
+
+    def test_dist_euclid(self):
+        dist = DistanceFunctions.dist_euclid((3, 0))
+        self.assertTrue(dist == 3)
+
+        dist = DistanceFunctions.dist_euclid((-3, 0))
+        self.assertTrue(dist == 3)
+
+        dist = DistanceFunctions.dist_euclid((-3, 0), (3, 0))
+        self.assertTrue(dist == 6)
```

Comment:
```suggestion
        dist = DistanceFunctions.dist_euclid((-3, 0), (1, 3))
        self.assertTrue(dist == 5)
```
Thought I had suggested this change on Lizzi's PR but maybe she didn't add it or this version is out of date?